<launch>
    <node pkg="turtlesim" type="turtlesim_node" name="turtlesim_node" />
    <node pkg="learning_ros" type="square.py"  name="my_node" />

</launch>